azurelinuxagent/common/protocol/wire.py

```diff
@@ -1453,6 +1453,12 @@ def parse(self, xml_text):
         data = findtext(xml_doc, "Data")
         if data is None:
             return
+    
+        # if the certificates format is not Pkcs7BlobWithPfxContents do not parse it
+        certificateFormat = findtext(xml_doc, "Format")
+        if certificateFormat and certificateFormat != "Pkcs7BlobWithPfxContents":
+            logger.verbose("The Format is not Pkcs7BlobWithPfxContents. Format is " + certificateFormat)
```

Comment:
Can you change this to WARN? Not having correct cert format may lead to extensions not working correctly.